pkg/apis/servicecatalog/types.go

```diff
@@ -145,10 +145,15 @@ type CommonServiceBrokerSpec struct {
 // ServiceClass allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
 //   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   externalID - the value set to [Cluster]ServiceClass.Spec.externalID
+//   serviceBrokerName - the value set to [Cluster]ServiceClass.Spec.serviceBrokerName
 //
 // ServicePlan allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
 //   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   externalID - the value set to [Cluster]ServiceClass.Spec.externalID
```

Comment:
The keys for these properties should be spec.externalName, spec.externalID, spec.serviceClass.name, and spec.clusterServiceClass.name.